src/frame/worker/crypto/recrypt/RecryptWasm.ts

```diff
@@ -95,6 +95,34 @@ export const generateSigningKeyPair = (): Future<Error, SigningKeyPair> => Futur
 export const getPublicSigningKeyFromPrivate = (privateSigningKey: PrivateKey<Uint8Array>) =>
     Future.tryF(() => RecryptApi.computeEd25519PublicKey(privateSigningKey));
 
+const isBufferAllZero = (bytes: Uint8Array) => {
+    return bytes.every((val) => val === 0);
+};
```

Comment:
Add a quick comment block explaining what this does. Also you could shorten this by just doing

```
const isBufferAllZero = (bytes: Uint8Array) => bytes.every((val) => val === 0);
```

Reply:
done